src/index.js

```diff
@@ -1,4 +1,9 @@
-import { factory, defaultConfig } from './Block';
-import { getDataProps } from './Block/data';
+import { createFactory } from '@volusion/element-block-utils';
+import { StarterBlockFactory } from './block';
+import { styles } from './styles';
```

Comment:
We also talked about updating the styles function (and the references in the createFactory function) to `getStyles()`

Reply:
@chris-decker-volusion 
Do you think we should update the filename as well?
i.e. `import { getStyles } from './styles';` vs `import { getStyles } from './getStyles';

Reply:
Yes